adhocteam/Head-Start-TTADP
# Add missing grants to reports

## Description of change

Adds script that adds missing grants to legacy reports. The IDs of the reports were found in production by searching for their display ID.

## How to test

 * Run tests
 * Run the script locally, note all the "missing grant" messages
 * Double check my mapping of display id to database id. Also verify CSV has correct database id.

|Display|Id|
|-|-|
|R02-AR-000726|312|
|R02-AR-000722|1020|
|R02-AR-000716|924|
|R02-AR-000671|925|
|R02-AR-000663|365|
|R02-AR-000650|346|

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-75

## Checklists

### Every PR


- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
